### DISCUSSION TITLE
Cant Locate Dark Mode on Linux

### DISCUSSION THREAD

**Discussion by JJW836412** [organicmaps/organicmaps#9101] [Q&A] [answered]
Idk if I'm dumb but iv done research and am unable to find dark mode for Linux.

I looked everywhere in the app
![Screenshot at 2024-08-25 18-23-46](https://github.com/user-attachments/assets/b517dfe3-621d-4d45-a310-0449796c57c1)

And it even shows screenshots in the download page
![Screenshot at 2024-08-25 18-27-22](https://github.com/user-attachments/assets/3c2bab12-0e7d-4069-8f93-1f54d015e677)

**Comment by rafalhogent** [comment]
It's still a dev feature..., From GNOME forum:

1. right-click on top bar menu near help
2. choose 'search'
3. in text-field type ` ?dark ` or  `?light`

![organic-dark](https://github.com/user-attachments/assets/d2ad1d65-9f49-4354-9fe4-cd446e9b89c5)

**Reply by JJW836412** [reply]
Thank you!

**Comment by map-per** [comment, marked as answer]
A new setting was added recently, so you can now switch do dark mode in the Preferences.
